src/le.py

```diff
@@ -2035,6 +2069,8 @@ def __init__(self):
         # Enabled fine-grained logging
         self.debug = False
         # All recognized events are logged
+        self.debug_cmds = False
```

Comment:
this is a debug option for logging what command is passed to the agent - hence I called it 'debug_cmds' - I'll have a think about it